### PR TITLE
feat(llc): added not exists filter

### PR DESCRIPTION
**Pull request by deven98**

### CODE REVIEW

File: packages/stream_chat/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Upcoming
+
+✅ Added
+
+- `Filter.notExists` now replaces `Filter.exists(key, exists: false)`.
```
Comment (imtoori) [inline packages/stream_chat/CHANGELOG.md:5]:
```suggestion
- Added `Filter.notExists`.
```

File: packages/stream_chat/lib/src/core/models/filter.dart [hunk 2]
```diff
@@ -40,9 +40,12 @@ enum FilterOperator {
   /// Matches values with the specified prefix.
   autoComplete,
 
-  /// Matches values that exist/don't exist based on the specified boolean value.
+  /// Matches values that exist.
```
Comment (imtoori) [inline packages/stream_chat/lib/src/core/models/filter.dart:43]:
```suggestion
  /// Matches values that exist/don't exist based on the specified boolean value.
```

File: packages/stream_chat/lib/src/core/models/filter.dart [hunk 3]
```diff
@@ -163,10 +166,13 @@ class Filter extends Equatable {
   factory Filter.autoComplete(String key, String text) =>
       Filter._(operator: FilterOperator.autoComplete, key: key, value: text);
 
-  /// Matches values that exist/don't exist based on the specified boolean value.
+  /// Matches values that exist.
```
Comment (imtoori) [inline packages/stream_chat/lib/src/core/models/filter.dart:169]:
```suggestion
  /// Matches values that exist/don't exist based on the specified boolean value.
```

**Review by imtoori — CHANGES_REQUESTED** [review]
a couple of changes
also add it to the docs please

File: packages/stream_chat/lib/src/core/models/filter.dart [hunk 4]
```diff
@@ -43,6 +43,9 @@ enum FilterOperator {
   /// Matches values that exist/don't exist based on the specified boolean value.
   exists,
 
+  /// Matches values that don't exist.
+  notExists,
+
```
Comment (xsahil03x) [inline packages/stream_chat/lib/src/core/models/filter.dart:48]: we don't really need this

Comment (deven98) [reply packages/stream_chat/lib/src/core/models/filter.dart:48]: Yeah fair enough, it was done in another way before, it changed to make it non breaking

**Pull request merged**